distribution/docs/src/main/resources/content/_plugins/nitf-post-process-plugin.adoc

```diff
@@ -0,0 +1,37 @@
+:type: plugin
+:status: published
+:title: NITF Post-Process Plugin
+:link: _nitf_post_process_plugin
+:plugintypes: postprocessplugin
+:summary: Creates and stores the NITF thumbnail and NITF overview images.
+
+The NITF Post-Process Plugin is an experimental replacement for the <<NITF_Render_Plugin,NITF Render Plugin>> that creates and stores the NITF thumbnail and NITF overview images.
```

Comment:
✏️ May be worth mentioning NITF ingest performance improvements over the other plugin.